src/modules/logger/logger.cpp

```diff
@@ -573,60 +573,70 @@ void Logger::add_common_topics()
 #endif
 
 	// Note: try to avoid setting the interval where possible, as it increases RAM usage
-	add_topic("actuator_controls_0", 100);
-	add_topic("actuator_controls_1", 100);
-	add_topic("actuator_outputs", 100);
-	add_topic("airspeed");
+	add_topic("actuator_controls_0", 50);
+	add_topic("actuator_controls_1", 50);
+	add_topic("actuator_outputs", 50);
+	add_topic("airspeed", 50);
 	add_topic("att_pos_mocap", 50);
-	add_topic("battery_status", 300);
+	add_topic("battery_status", 10);
 	add_topic("camera_capture");
 	add_topic("camera_trigger");
-	add_topic("commander_state", 100);
+	add_topic("commander_state", 10);
```

Comment:
As a side note for the commander refactor I was thinking of having all commander publications at a very low rate, or immediate if changed.